src/main/scala/rocket/icache.scala

```diff
@@ -128,7 +128,7 @@ class ICache(latency: Int)(implicit p: Parameters) extends CoreModule()(p) with
       io.resp.bits.datablock := Mux1H(s1_tag_hit, s1_dout)
       io.resp.valid := s1_hit
     case 2 =>
-      val s2_hit = RegEnable(s1_hit, !stall)
+      val s2_hit = RegEnable(s1_hit, Bool(false), !stall)
```

Comment:
I'll manually snag the output port thing and put it in my next PR.

The formatting change is actually going in the opposite direction I've been trying for; trying to line everything up results in large commit diffs when something longer gets added.  (For the same reason, I didn't change the old formatting unless I was also changing the code for some other reason.)